docs/index.rst

```diff
@@ -0,0 +1,109 @@
+.. telemetry documentation master file, created by
+   sphinx-quickstart on Fri Sep 27 16:34:00 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Jupyter Telemetry
+=================
+
+**Configurable event-logging for Jupyter applications and extensions.**
+
+
+Telemetry provides a configurable traitlets object, EventLog, for structured event-logging in Python. It leverages Python's standard logging library for filtering, handling, and recording events. All events are validated (using jsonschema) against registered JSON schemas.
+
+If you're looking for telemetry in Jupyter frontend applications (like JupyterLab), checkout the work happening in jupyterlab-telemetry_! 
+
+.. _jupyterlab-telemetry: https://github.com/jupyterlab/jupyterlab-telemetry
```

Comment:
@jaipreet-s Great idea. What do you think of this line on the front page of the docs?

Reply:
SGTM!